Added Project Section

wajahatalli opened the pull request:
- Added Grid Layout
- Added Flexbox to align the cards.
- Used lists on navigation links.
- Implemented buttons interactions ( on hover, on press )
- Added project and about section
- Fixed image cards
- Fixed colors
- Design is matching the template
- Removed extra h1 tag

Kamba56 commented on index.html:
```diff
@@ -88,5 +88,171 @@ <h1>
+            sign-ups required.
+          </p>
+          <div class="tags">
+            <ul>
+              <li><a href="" class="tag">html</a></li>
+              <li><a href="" class="tag">css</a></li>
+              <li><a href="" class="tag">javascript</a></li>
+            </ul>
+          </div>
+          <button class="btn" type="button">See Project</button>
+        </div>
+      </div>
+      <div class="second_project">
+        <img
+          class="Tonic"
+          src="assets/images/Snapshoot Portfolio.png"
+          alt="Snapshoot"
+        />
+        <h2>Multi-Post Stories</h2>
+        <div class="flex_list">
+          <ul class="list">
+            <h5>CANOPY</h5>
+
+            <li class="list_items">Back End Dev</li>
+            <li class="list_items">2015</li>
+          </ul>
+          <p class="project_description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="tags">
+            <ul>
+              <li><a href="" class="tag">html</a></li>
+              <li><a href="" class="tag">css</a></li>
+              <li><a href="" class="tag">javascript</a></li>
+            </ul>
+          </div>
+          <button class="btn" type="button">See Project</button>
+        </div>
+      </div>
```
- [ ] Please fix the images in the project cards to match the design. Please edit the border color on the cards to match the design. Please increase the spacing between the cards to match the design. Kindly fix these please as the project requirements demanded that you must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- [ ] The project section on the design template has a background color, kindly add that to yours too, please.
###Your design
![1](https://user-images.githubusercontent.com/94551484/198819155-844f507d-53d0-45c0-81e2-254b8edfe5c6.png)

### Figma template 
![f1](https://user-images.githubusercontent.com/94551484/198819156-d1cd5058-2140-45d2-a611-29001032dcb7.png)

Kamba56 commented on index.html:
```diff
@@ -88,5 +88,171 @@ <h1>
+      </div>
+    </section>
+    <section class="About_Section">
+      <h1>About <br />Myself</h1>
```
- [ ] Please note that you should have only one `h1` in the whole of your document, kindly change this to any other heading tag(from `h2` to `h5`). This is SEO best practice. Kindly check [here](https://yoast.com/one-h1-heading-per-post/) for more information.

Kamba56 commented on index.html:
```diff
@@ -88,5 +88,171 @@ <h1>
       </div>
     </section>
 
+    <section class="project_section">
```
- [ ] Please use CSS grid to layout contents of the work section as demanded by the project requirements.

Kamba56 reviewed (changes requested):
Hi @wajahatalli, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] All linters passed!👏👏👏
- [x] Great UI design!👏👏👏
- [x] Great use of Github flow!👏👏👏
- [ ] Professional readme!👏👏👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

wajahatalli commented:
- Added Grid Layout
- Added Flexbox to align the cards.
- Used lists on navigation links.
- Implemented buttons interactions ( on hover, on press )
- Added project and about section
- Fixed image cards
- Fixed colors
- Design is matching the template
- Removed extra h1 tag

Favourezeugwa reviewed (approved):
## STATUS: APPROVED 🥇 🏆 
Hi @wajahatalli , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉  🎉 🎉

![happy](https://user-images.githubusercontent.com/99246359/198823303-b2db2a13-a046-4b29-9b8e-1cbd18972b16.gif)

## Highlights 💯 
- No linter error ✔️ 
- Good commit messages ✔️ 
- Descriptive pull request ✔️ 
- Used Github flow ✔️ 
- Professional Readme file ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
